examples/spark2/src/main/scala/org/apache/carbondata/examples/CompareTest.scala

```diff
@@ -306,6 +307,8 @@ object CompareTest {
     // do GC and sleep for some time before running next table
     System.gc()
     Thread.sleep(1000)
+    System.gc()
```

Comment:
There is no guarntee that GC will be called after calling of System.gc(), thats why after waiting for 1 second called again to increase the probability of running GC